tests/test_smoke.py

```
@@ -61,6 +61,24 @@ def test_main_defaults_to_streamable_http(_mock_mcp_run):
+    with (
+        patch("sys.argv", ["okp-mcp"]),
+        patch.dict("os.environ", {"MCP_GLITCHTIP_DSN": "https://glitchtip.example.com/1"}),
+        patch("okp_mcp.initialize_error_reporting") as initialize_error_reporting,
+    ):
+        main()
+
+    initialize_error_reporting.assert_called_once()
+    glitchtip_dsn = initialize_error_reporting.call_args.args[0].glitchtip_dsn
+    assert glitchtip_dsn is not None
+    assert glitchtip_dsn.get_secret_value() == "https://glitchtip.example.com/1"
+    _assert_http_run(_mock_mcp_run, transport="streamable-http", host="0.0.0.0", port=8000)
```

Ruff (0.15.11)

[error] Possible binding to all interfaces

(S104)